libraries/CMakeLists.txt

```diff
@@ -45,3 +45,10 @@ set_property(GLOBAL PROPERTY CTEST_CUSTOM_TESTS_IGNORE
   attest pbkdf2 parsing ${_CTEST_CUSTOM_TESTS_IGNORE}")
 
 add_subdirectory( rodeos )
+
+message(STATUS "Starting AMQP-CPP configuration...")
+find_package(OpenSSL REQUIRED)
+set(AMQP-CPP_LINUX_TCP ON)
```

Comment:
The build seems to be failing, and cmake warns
```
CMake Warning (dev) at libraries/amqp-cpp/CMakeLists.txt:24 (option):
  Policy CMP0077 is not set: option() honors normal variables.  Run "cmake
  --help-policy CMP0077" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  For compatibility with older versions of CMake, option is clearing the
  normal variable 'AMQP-CPP_LINUX_TCP'.
This warning is for project developers.  Use -Wno-dev to suppress it.
```
seems you'll need to replace the `set()` with appropriate `option()` instead